project/spotify/static/spotify/neighbors.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	
+	$.getJSON("/neighborsinfo",function(data) {
+		var users = data['user_list']
+		var neighbors = data['neighbors_list']
+		for (user_index in users) {
+			$("#neighbors").append(users[user_index] +"<ol>")
+			for (neighbor in neighbors[user_index]) {
+				$("#neighbors").append("<li>"+neighbors[user_index][neighbor] + "</li>")
```

Comment:
Line is too long.<br>Identifier 'user_index' is not in camel case.<br>Missing semicolon.<br>'user_index' is not defined.<br>'neighbor' is not defined.